docs/.templates/integration/schema.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories_list": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
```

Comment:
Maybe we can add an `id` and a `parent` props here in order to handle the recursive nature of categories/subcategories thing.

For example, `Sub category` belongs to `Top level category` here:
```
// Top level category
{
  id: 1,
  parent: null,
  name: "Top level category"
}

// Sub category
{
  id: 2,
  parent: 1,
  name: "Sub category"
}

Reply:
@kapantzak nice idea, just a question, where would the `id` field be used? Do we need it in the app somewhere? Is it going to be the order that things show up?

Reply:
We also need a priority field, but we can use priority as an ID too. Commit incoming